[2024-05-31T11:00:37.677+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-31T11:00:11.907917+00:00 [queued]>
[2024-05-31T11:00:37.685+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-31T11:00:11.907917+00:00 [queued]>
[2024-05-31T11:00:37.686+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-05-31T11:00:37.686+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-05-31T11:00:37.686+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-05-31T11:00:37.698+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): set_up_database> on 2024-05-31 11:00:11.907917+00:00
[2024-05-31T11:00:37.702+0000] {standard_task_runner.py:55} INFO - Started process 1632 to run task
[2024-05-31T11:00:37.704+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'conference_dag', 'set_up_database', 'manual__2024-05-31T11:00:11.907917+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/conference_dag.py', '--cfg-path', '/tmp/tmpb0_2hum9']
[2024-05-31T11:00:37.705+0000] {standard_task_runner.py:83} INFO - Job 23: Subtask set_up_database
[2024-05-31T11:00:37.754+0000] {task_command.py:388} INFO - Running <TaskInstance: conference_dag.set_up_database manual__2024-05-31T11:00:11.907917+00:00 [running]> on host 312e62f5d623
[2024-05-31T11:00:37.810+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=conference_dag
AIRFLOW_CTX_TASK_ID=set_up_database
AIRFLOW_CTX_EXECUTION_DATE=2024-05-31T11:00:11.907917+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-31T11:00:11.907917+00:00
[2024-05-31T11:00:37.811+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-05-31T11:00:37.812+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'set -e; python /opt/***/mysql_utils/set_up_database.py']
[2024-05-31T11:00:37.819+0000] {subprocess.py:86} INFO - Output:
[2024-05-31T11:00:38.178+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-31T11:00:38.197+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=conference_dag, task_id=set_up_database, execution_date=20240531T110011, start_date=20240531T110037, end_date=20240531T110038
[2024-05-31T11:00:38.237+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-05-31T11:00:38.260+0000] {taskinstance.py:2578} INFO - 3 downstream tasks scheduled from follow-on schedule check
